<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsageStatistic::Internal::UsageStatisticSimpleWidget</class>
 <widget class="QWidget" name="UsageStatistic::Internal::UsageStatisticSimpleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="toggleRow"/>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="plainText">
      <string>Gathers data on user interactions, events, crash reports, and the usage of various application components. This data is stored both locally and on Qtâ€™s cloud, where it is analysed to facilitate improvements in future versions of Qt Design Studio.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblPrivacyPolicy">
     <property name="text">
      <string>Legal Notice and Privacy Policy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
